apps/stripe/src/app/api/stripe/webhook/use-case.ts

```diff
@@ -184,6 +187,14 @@ export class StripeWebhookUseCase {
         );
 
         if (reportResult.isErr()) {
+          if (reportResult.error instanceof TransactionEventReporterErrors.AlreadyReportedError) {
+            this.logger.info("Transaction event already reported", {
+              error: reportResult,
```

Comment:
Not sure if we should attach error. It may confuse datadog, and this is not error actually